web/src/app/admin/admin.component.scss

```diff
@@ -0,0 +1,74 @@
+@import '../../styles.scss';
+@import '../../assets/scss/variable';
+
+.users {
+  padding: 20px 30px;
+}
+
+.home {
```

Comment:
Is `home` the right name here?

Reply:
okay will create users-list component and move everything there